test/test_image_streamer_build_plan.py

```diff
@@ -23,6 +23,34 @@
 
 FAKE_MSG_ERROR = 'Fake message error'
 
+BUILD_PLAN_CREATE = dict(
+    config='config.json',
+    api_version=600,
```

Comment:
Do we need to give api_version explicitly as it's already present in config file.